dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifierRepositoryImpl.java

```diff
@@ -0,0 +1,118 @@
+package de.digitalcollections.cudami.server.backend.impl.jdbi.identifiable;
+
+import de.digitalcollections.cudami.server.backend.api.repository.identifiable.IdentifierRepository;
+import de.digitalcollections.cudami.server.backend.impl.jdbi.AbstractPagingAndSortingRepositoryImpl;
+import de.digitalcollections.model.api.identifiable.Identifier;
+import de.digitalcollections.model.api.paging.PageRequest;
+import de.digitalcollections.model.api.paging.PageResponse;
+import de.digitalcollections.model.impl.identifiable.IdentifiableImpl;
+import de.digitalcollections.model.impl.identifiable.IdentifierImpl;
+import de.digitalcollections.model.impl.paging.PageResponseImpl;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import org.jdbi.v3.core.Jdbi;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public class IdentifierRepositoryImpl extends AbstractPagingAndSortingRepositoryImpl implements IdentifierRepository {
```

Comment:
Warum denn ein `extends` hier? Alle Methoden in der abstrakten Klasse könnten statisch sein, dann könnte man das in eine `PageHelper` Utility-Klasse umziehen und hier auf die Vererbung verzichten.

Reply:
wie besprochen lassen wir dies so